clients/snowflake/staging.go

```diff
@@ -115,7 +115,7 @@ func (s *Store) mergeWithStages(ctx context.Context, tableData *optimization.Tab
 		return nil
 	}
 
-	fqName := tableData.ToFqName(ctx, constants.Snowflake, true)
+	fqName := tableData.ToFqName(ctx, s.Label(), true, "")
```

Comment:
I assume this is the same thing as with BigQuery, but just calling it out.

Reply:
yup!